README.md

```diff
@@ -43,9 +43,11 @@ For more information on [how to get involved on the CHAOSS website](https://chao
 
 ## Collecting Data
 
-Augur supports ```Python3.7``` through ```Python3.11``` on all platforms. ```Python3.12``` and above do not yet work because of machine learning worker dependencies. On OSX, you can create a ```Python3.11``` environment, by running:
+Augur officially supports ```Python3.11``` on all platforms. ```Python3.12``` and above do not yet work because of machine learning worker dependencies. We recommend using [uv](https://github.com/astral-sh/uv) to manage your Python environment:
```

Comment:
I think we should just mention the official policy that Augur aims to work with all supported python versions, although we are dependent on support from our dependencies so new versions may not be fully supported right out of the gate

Reply:
I think I prefer the phrasing about python versions from https://github.com/chaoss/augur/pull/3448